src/types/base.ts

```diff
@@ -105,3 +105,12 @@ export interface ReportOpts {
  *   loading. This is equivalent to the corresponding `readyState` value.
  */
 export type LoadState = 'loading' | 'dom-interactive' | 'dom-content-loaded' | 'complete';
+
+/**
+ * Extend the document with the new wasDiscarded boolean which is not supported
+ * in typescript yet
+ * https://github.com/WICG/page-lifecycle/blob/main/README.md
+ */ 
+declare global {
```

Comment:
This is fine, but can you move it here, similar to `document.prerendering`? https://github.com/GoogleChrome/web-vitals/blob/7f0ed0bfb03c356e348a558a3eda111b498a2a11/src/types.ts#L61-L64

Reply:
That took me ages to figure out! If only I'd looked harder and found this one!